api/src/utils/2d-map.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+const sharp = require('sharp');
+
+const w_thumb = 400;
+const getSvgThumbnail = async (svgName, model) => {
+  const sharp = require('sharp');
+  const svgFile = `/project/svg/${model}/${svgName}.svg`;
+  const svgThumbnail = await sharp(svgFile)
+    .resize(w_thumb)
+    .flatten({ background: { r: 255, g: 255, b: 255 } })
+    .png()
+    .toBuffer()
+    .then(function(info) {
+      console.log(info)
+      return info
+    })
+    .catch(function(err) {
+      console.log(err)
```

Comment:
We will likely want to remove these `console.log` before merging, so I'm just making a note of them.

Reply:
These lines are removed with the commit b6399bb7